json/adguard.json

```
@@ -30,5 +30,10 @@
         "username": null,
         "password": null
     },
-    "notes": []
-}
\ No newline at end of file
+    "notes": [
+        {
+            "text": "AdguardHome is only updatable via the WebUI!",
```

```suggestion
            "text": "AdguardHome is only updatable via the WebUI!",
```
Think having this extra note in documentation is quite verbose, is it not enough to get the info when running update?

---

If we do want it I would change it to similar message as above: `Adguard Home should be updated via the user interface`